### PR TITLE
Rgb image adapt

### PR DESCRIPTION
**Pull request by davidpiuva**
Adapting things to RGB instead of BGR.

### CODE REVIEW

**Comment by davidpiuva** [comment]
The last commit will have to be rewritten from scratch.

**Comment by davidpiuva** [comment]
Cherry-picked to handle version conflict.

**Comment by davidpiuva** [comment]
@thomasfanell Peer review?

File: test/draw/RasterBgrTest.ooc [hunk 1]
```diff
@@ -1,120 +0,0 @@
...
- * This software may be modified and distributed under the terms
- * of the MIT license.  See the LICENSE file for details.
- */
-
-use base
-use draw
-use geometry
-use unit
-
-RasterBgrTest: class extends Fixture {
```
Comment (davidpiuva) [inline test/draw/RasterBgrTest.ooc]: The file was renamed, not deleted.

File: test/draw/RasterBgraTest.ooc [hunk 2]
```diff
@@ -1,77 +0,0 @@
...
- * This software may be modified and distributed under the terms
- * of the MIT license.  See the LICENSE file for details.
- */
-
-use base
-use draw
-use geometry
-use unit
-
-RasterBgraTest: class extends Fixture {
```
Comment (davidpiuva) [inline test/draw/RasterBgraTest.ooc]: The file was renamed, not deleted.

File: test/draw/RasterMonochromeTest.ooc [hunk 3]
```diff
@@ -118,11 +118,11 @@ RasterMonochromeTest: class extends Fixture {
 			expect(image[0, 2] y as Int, is equal to(value as Int))
 			image referenceCount decrease()
 		})
-		/*this add("distance, convertFrom RasterBgra", func {
+		/*this add("distance, convertFrom RasterRgba", func {
```
Comment (thomasfanell) [inline test/draw/RasterMonochromeTest.ooc]: Perhaps not for this PR, but are we expecting to uncomment this anytime soon?

Comment (davidpiuva) [inline test/draw/RasterMonochromeTest.ooc]: No idea.

**Comment by davidpiuva** [comment]
@fredrikbryntesson Time to merge?

**Pull request merged**